### PR TITLE
Added blog for installation ceremony

### PR DESCRIPTION
**Pull request by Saptashrungi**
The blog for installation ceremony has been added.

### CODE REVIEW

**Comment by Saptashrungi** [comment]
@czgdp1807  both PRs have been successfully done.

**Comment by czgdp1807** [comment]
I think you have added the blog on top of the minutes of meetings commit. 
To fix this, may be drop https://github.com/rtrclubiitj/rtrclubiitj.github.io/pull/37/commits/c07081ece76b9f79045fb9f58a5463d1f7855fca.
You can try to re-base in interactive mode.
For future,
Please go back to master and then checkout to a new branch for starting new work.

File: _posts/2019-09-27-post_18.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+date:   2019-09-18
+author: ""
+excerpt: "The Rotaract club of IIT Jodhpur successfully celebrated its motivational and encouraging Installation Ceremony."
+image: "/images/post_18.jpg"
+is_pinned: false
+---
+
+
+
+
```
Comment (czgdp1807) [inline _posts/2019-09-27-post_18.md]: Remove L10-L13.

**Comment by czgdp1807** [comment]
When are you planning to computer this PR?

**Comment by Saptashrungi** [comment]
I will edited the mom2 file by adding yaml

**Comment by czgdp1807** [comment]
You have to address [this comment](https://github.com/rtrclubiitj/rtrclubiitj.github.io/pull/37#issuecomment-535890436).

**Comment by Saptashrungi** [comment]
I am not getting how to drop comment and didn't get rebasing to interactive
mode

**Comment by czgdp1807** [comment]
@asharani97 Please make a new PR for this one too. Thanks.

**Pull request closed without merging**